app/src/main/java/com/github/ybecker/epforuml/AnswerAdapter.kt

```diff
@@ -1,14 +1,20 @@
 package com.github.ybecker.epforuml
 
+import android.app.Activity
+import android.content.Intent
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
 import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat.startActivity
 import androidx.recyclerview.widget.RecyclerView
+import com.github.ybecker.epforuml.database.DatabaseManager
 import com.github.ybecker.epforuml.database.DatabaseManager.db
 import com.github.ybecker.epforuml.database.Model
 
-class AnswerAdapter(private val questionId : String, private val questionText : String, private val answerList : List<String>)
+class AnswerAdapter(private val questionId : String, private val questionText : String, private val answerList : List<String>,private val mainActivity: Activity)
```

Comment:
Wasn't it possible to use the ApplicationContext instead of adding the activity as an argument ?